lte/gateway/c/connection_tracker/src/EventTracker.cpp

```diff
@@ -11,28 +11,23 @@
  * limitations under the License.
  */
 
+#include <arpa/inet.h>
+#include <glog/logging.h>
+#include <libmnl/libmnl.h>
+#include <linux/netfilter/nfnetlink_compat.h>
+#include <linux/netlink.h>
+#include <netinet/in.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <unistd.h>
-#include <netinet/in.h>
-#include <arpa/inet.h>
 #include <iostream>
-
-#include <libmnl/libmnl.h>
 #include <linux/netfilter/nfnetlink.h>
 #include <linux/netfilter/nfnetlink_conntrack.h>
 
-#include <linux/if_packet.h>
-#include <string.h>
-#include <sys/ioctl.h>
-#include <sys/socket.h>
-#include <net/if.h>
-#include <netinet/ether.h>
-#include <linux/ip.h>
-#include <memory>
-
 #include "lte/gateway/c/connection_tracker/src/EventTracker.hpp"
```

Comment:
Should `EventTracker.hpp` be located at top because it is the own header interface?

Reply:
hmm yes that is a good question.. It seems like IWYU formats it correctly for some files but not all. I will fiddle with configs to see if I can get it right. (The changes are all automated, so I don't want to manually fix anything)

Reply:
Was able to automate the fix for this by running fix_includes.py --sort_only -s lte/gateway/c/**/*.cpp

Reply:
actually it fixed other instances but not this one :o

Reply:
fixed